### ISSUE TITLE
[Feature Request] Add support for Stage channels.

### ISSUE THREAD

**Issue by tlworks** [jagrosh/MusicBot#655]
**Is your feature request related to a problem? Please describe.**
Bot cannot join the newly released stage channels.

**Describe the solution you'd like**
Add support for stage channels.

**Additional context**
None

**Comment by AngelKotik** [comment]
[https://github.com/jagrosh/MusicBot/discussions/657](._.)

**Comment by nrvnzsmbruch** [comment]
The stage channel is for podcast stuff, not music?

**Comment by Thurrax** [comment]
> The stage channel is for podcast stuff, not music?

Voice Channels are for talking aint they? Also i can think of easier Management with Staged Channels without setting all Roles and Rights.

**Comment by Sanduhr32** [comment]
Support for Stage Channels will come soon.
Probably next update will feature newer versions of fundamental libraries granting support you are asking for.

**Comment by AngelKotik** [comment]
![Discord_YhdWiAJbDm](https://user-images.githubusercontent.com/46861834/114430540-354d7d00-9bc7-11eb-902d-58f2a9ce8cc7.png)
cool

**Comment by epicmario7133** [comment]
waiting for update...

**Comment by eltaylor1104** [comment]
It's already possible, though with a few bugs. Join a channel, play a song, and use the discord "move to" feature to move it to a Stage Channel. Make sure it has "Stage Moderator" permissions, and click "invite to speak." the bot will play your music. The only bug is that to play a song, you need to join the original channel that you moved the bot from, as the bot still thinks it is in there.

**Comment by tlworks** [comment]
> It's already possible, though with a few bugs. Join a channel, play a song, and use the discord "move to" feature to move it to a Stage Channel. Make sure it has "Stage Moderator" permissions, and click "invite to speak." the bot will play your music. The only bug is that to play a song, you need to join the original channel that you moved the bot from, as the bot still thinks it is in there.

This is a workaround, not a feature.

**Comment by eltaylor1104** [comment]
> > It's already possible, though with a few bugs. Join a channel, play a song, and use the discord "move to" feature to move it to a Stage Channel. Make sure it has "Stage Moderator" permissions, and click "invite to speak." the bot will play your music. The only bug is that to play a song, you need to join the original channel that you moved the bot from, as the bot still thinks it is in there.
> 
> 
> 
> This is a workaround, not a feature.
Yes, I never claimed it was a feature. But, until support for it is released, which is not right now, that will work for my server. I keep it in a stage channel 24/7. I wouldn't be using that method if there was support for stage channels lol.

**Comment by benCofficial** [comment]
Bump! Is this supported yet? (I’m slow)

**Comment by aspwil** [comment]
bump! with youtube now taking down bots I'm hoping for this feature as my server likes to host public radio-type deals for people to listen to.

**Comment by testbot-github** [comment]
Bump! Is this supported yet?

**Comment by MichailiK** [comment]
> Bump! Is this supported yet?

If it were, this issue would be closed.

**Comment by JackeLeMort** [comment]
i'm currently listening to bot in a stage channel so yeah, but i can't set the channel by default
